test/functional/apps/settings/_index_pattern_results_sort.js

```diff
@@ -93,9 +93,11 @@ define(function (require) {
         });
 
         bdd.it('makelogs data should have expected number of fields', function () {
-          return settingsPage.getFieldsTabCount()
-          .then(function (tabCount) {
-            expect(tabCount).to.be('' + expectedFieldCount);
+          return common.tryForTime(5000, function () {
```

Comment:
I really don't like this in here, it should be in the SettingsPage helper. As it is, the `setFindTimeout` is waiting for 60 seconds in there, why is this 5 seconds. Where did that number come from? Do we need to do that every time we use `getFieldsTabCount`?

This should just happen in `getFieldsTabCount`. Maybe put a `tryForTime` using `defaultTimeout` in there, and make `setFindTimeout` something like `defaultTimeout / 3`, so it'll try 3 times before failing. What do you think about that?